cmake/modules/StandaloneCrt.cmake

```diff
@@ -15,41 +15,29 @@
 # specific language governing permissions and limitations
 # under the License.
 
-
 if(USE_MICRO)
 
-if(MSVC)
-
-  # When building for Windows, use standard CMake for compatibility with
-  # Visual Studio build tools and not require Make to be on the system.
-
-  # TODO: test building with this MSVC conditional code removed
-  # when USE_MICRO is enabled
+  message(STATUS "Build microTVM RPC common")
 
-  set(CRT_CONFIG, "src/runtime/micro/crt_config.h")
+  # add microTVM RPC common files to TVM runtime build
+  list(APPEND TVM_CRT_SOURCES
+      3rdparty/libcrc/src/crcccitt.c
+      src/runtime/crt/microtvm_rpc_common/frame_buffer.cc
+      src/runtime/crt/microtvm_rpc_common/framing.cc
+      src/runtime/crt/microtvm_rpc_common/session.cc
+      src/runtime/crt/microtvm_rpc_common/write_stream.cc)
 
-  add_library(host_standalone_crt
-              STATIC
-              3rdparty/libcrc/src/crcccitt.c
-              src/runtime/crt/microtvm_rpc_common/frame_buffer.cc
-              src/runtime/crt/microtvm_rpc_common/framing.cc
-              src/runtime/crt/microtvm_rpc_common/session.cc
-              src/runtime/crt/microtvm_rpc_common/write_stream.cc)
+  list(APPEND RUNTIME_SRCS ${TVM_CRT_SOURCES})
+  include_directories(SYSTEM src/runtime/micro)
```

Comment:
Why the `SYSTEM` here?

Reply:
it needs a header in that directory to build those files that are in TVM_CRT_SOURCES, it's the same pattern used in other .cmake and CmakeLists.txt